crates/test-assembly/crates/test_ns_string/lib.rs

```diff
@@ -6,14 +6,14 @@ use icrate::Foundation::NSString;
 
 // Temporary to allow testing putting string references in statics.
 // This doesn't yet compile on other platforms, but could in the future!
-#[cfg(feature = "apple")]
+#[cfg(all(feature = "apple", feature = "unstable-static-nsstring"))]
```

Comment:
Use `"assembly-features"` instead, since `"unstable-static-nsstring"` is not defined in `test_ns_string/Cargo.toml`